lte/gateway/configs/spgw.yml

```diff
@@ -57,4 +57,4 @@ agw_l3_tunnel: false
 ovs_gtpu_checksum: false
 
 # Enable IPv6 support for S1AP SCTP endpoint
-s1_ipv6_enabled: false
+s1_ipv6_enabled: true
```

Comment:
this looks like not used anywhere (s1ap_ipv6_enabled) in mme.yml looks like the universal config. You can remove this completely I think.

Reply:
or is the intent to enable/disable CP and UP IPv6 paths separately?

Reply:
right. this allows separate control over CP and UP over IPv6.

Reply:
but the current code generation does not use this at all. check the template files.

Reply:
it is there. checkout MME_S1_IPV6_ENABLED and SGW_S1_IPV6_ENABLED.

Reply:
please check what is used in spgw.template.conf and mme.template.conf and spgw.template.conf:
`MME_S1_IPV6_ENABLED                   = "{{ s1ap_ipv6_enabled }}";`
`SGW_S1_IPV6_ENABLED = "{{ s1ap_ipv6_enabled }}";`

Reply:
they both use the same flag.

Reply:
Internally these are two different features, so for me it makes sense to have separate flag on AGW to protect each feature.

At user API level we can have single switch to enable IPv6, But I am fine with unifying it even at AGW level.

I think it would it would be tricky to unify these two configurations in single yml file due to existing config management in spgw and mme. @koolzz can you confirm?

Reply:
> 

typo in template for SGW_S1_IPV6_ENABLED needs to be fixed.

Reply:
fixed it